### DISCUSSION TITLE
Start on boot, system tray option

### DISCUSSION THREAD

**Discussion by brandon14** [ente-io/ente#2068] [Enhancements]
It would be neat to add options to run Ente Auth on start-up, as well as give the option to run in the system tray and minimize to the system tray. I am not sure how difficult this would be or if it's even possible to make cross-platform.

**Comment by gitmeED331** [comment]
Yes, this is needed. 

The current behavior is terrible. If I close the window (clicking the X or killactive) it closes the application entirely, who said I wanted to that? I just was trying to get rid of the window, not the app itself. The app should have an option to "close-to-tray".

There also needs to have "start-in-tray" option, where on start-up the app window does not open. Current behavior is intrusive, annoying, and a security risk.

OS: Arch Linux
Desktop: Hyprland
